demoproviders/barbican/java/org/openyolo/demoprovider/barbican/provider/DeleteCredentialActivity.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2017 The OpenYOLO Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.openyolo.demoprovider.barbican.provider;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.annotation.UiThread;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.View.OnClickListener;
+import android.view.WindowManager.LayoutParams;
+import android.widget.Button;
+import android.widget.TextView;
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import java.io.IOException;
+import java.util.Set;
+import org.openyolo.demoprovider.barbican.R;
+import org.openyolo.demoprovider.barbican.storage.CredentialStorageClient;
+import org.openyolo.demoprovider.barbican.storage.CredentialStorageClient.ConnectedCallback;
+import org.openyolo.protocol.AuthenticationDomain;
+import org.openyolo.protocol.Credential;
+import org.openyolo.protocol.CredentialDeleteRequest;
+import org.openyolo.protocol.CredentialDeleteResult;
+import org.openyolo.protocol.MalformedDataException;
+import org.openyolo.protocol.Protobufs;
+
+public class DeleteCredentialActivity extends AppCompatActivity implements ConnectedCallback {
+
+    private static final String LOG_TAG = "DeleteCrActivity";
```

Comment:
I'm paranoid about long log tag names after being burned by the length restriction in the past. I just wanted something close enough to the activity name that you would be able to find it.